.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "\U0001F680Feature request"
+about: I have a suggestion (and might want to implement myself)
+title: ''
+labels: enhancement, needs triage
+assignees: ''
+---
+
+<!-- Consider opening a pull request instead: it’s a more productive way to discuss new features -->
```

Comment:
I don't totally agree with this, it means that someone might spend a lot of time implementing a feature only to be told, "no this doesn't make sense in emotion".
```suggestion
```